scripts/updateProjectReferences.ts

```diff
@@ -144,5 +144,37 @@ import { getPrettierConfig } from './utils/getPrettierConfig';
                     process.exit(1);
                 }
             }
+
+            // Copy references also to tsconfig.libESM.json if exists
```

Comment:
You shoudn't need this, just extend `tsconfig.lib.json` in root `tsconfig.libESM.json` instead of `base` file

Reply:
Ok, so it sounds like a good idea. Do you mean extending the `tsconfig.lig.json` from the package and just changing the target like here https://github.com/trezor/trezor-suite/pull/16470/files#diff-97fc10c343bde083841616cc3853cf0d7e98003d6a1b11aa36a7665e69678b0b  ?